### ISSUE TITLE
conda update conda error - win 10 - all users

### ISSUE THREAD

**Issue by codedocta** [conda/conda#4613]
Hi,

This is an fresh install on win 10 14393 64bit of Anaconda.
Installed on an Admin account. Intended for additional users but I have yet to make another account. No other Python installs on this machine. No other errors during the Anaconda3 install.

Hope this helps,
Nick

```
(C:\ProgramData\Anaconda3) C:\Users\kivy-donor>conda update conda
Fetching package metadata ...An unexpected error has occurred.
Please consider posting the following information to the
conda GitHub issue tracker at:

    https://github.com/conda/conda/issues



Current conda install:

               platform : win-64
          conda version : 4.3.8
       conda is private : False
      conda-env version : 4.3.8
    conda-build version : not installed
         python version : 3.6.0.final.0
       requests version : 2.12.4
       root environment : C:\ProgramData\Anaconda3  (writable)
    default environment : C:\ProgramData\Anaconda3
       envs directories : C:\ProgramData\Anaconda3\envs
          package cache : C:\ProgramData\Anaconda3\pkgs
           channel URLs : https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/win-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
            config file : None
           offline mode : False
             user-agent : conda/4.3.8 requests/2.12.4 CPython/3.6.0 Windows/10 Windows/10.0.14393



`$ C:\ProgramData\Anaconda3\Scripts\conda-script.py update conda`




    Traceback (most recent call last):
      File "C:\ProgramData\Anaconda3\lib\site-packages\conda\exceptions.py", line 617, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "C:\ProgramData\Anaconda3\lib\site-packages\conda\cli\main.py", line 137, in _main
        exit_code = args.func(args, p)
      File "C:\ProgramData\Anaconda3\lib\site-packages\conda\cli\main_update.py", line 65, in execute
        install(args, parser, 'update')
      File "C:\ProgramData\Anaconda3\lib\site-packages\conda\cli\install.py", line 210, in install
        unknown=index_args['unknown'], prefix=prefix)
      File "C:\ProgramData\Anaconda3\lib\site-packages\conda\core\index.py", line 120, in get_index
        index = fetch_index(channel_priority_map, use_cache=use_cache)
      File "C:\ProgramData\Anaconda3\lib\site-packages\conda\core\index.py", line 445, in fetch_index
        repodatas = _collect_repodatas(use_cache, urls)
      File "C:\ProgramData\Anaconda3\lib\site-packages\conda\core\index.py", line 433, in _collect_repodatas
        repodatas = _collect_repodatas_serial(use_cache, urls)
      File "C:\ProgramData\Anaconda3\lib\site-packages\conda\core\index.py", line 401, in _collect_repodatas_serial
        for url in urls]
      File "C:\ProgramData\Anaconda3\lib\site-packages\conda\core\index.py", line 401, in <listcomp>
        for url in urls]
      File "C:\ProgramData\Anaconda3\lib\site-packages\conda\core\index.py", line 141, in func
        res = f(*args, **kwargs)
      File "C:\ProgramData\Anaconda3\lib\site-packages\conda\core\index.py", line 391, in fetch_repodata
        with open(cache_path, 'w') as fo:
    PermissionError: [Errno 13] Permission denied: 'C:\\ProgramData\\Anaconda3\\pkgs\\cache\\2116b818.json'
```

**Comment by codedocta** [comment]
However, 

I ran it "Run as Administrator" from the task bar(Anaconda prompt) and all went well. But again I am on the only account on this machine with a password even.

**Comment by lgjiao** [comment]
Also new install, Win10, x64 version.
I got a similar message when I ran 'conda update anaconda'. 'Run as Administrator' indeed solve this problem.

Use of 'Anaconda Navigator' is suggested and does not have such problem.

**Comment by codedocta** [comment]
In addition did not add Anaconda3 to PATH.

**Comment by ijstokes** [comment]
@kalefranz how does someone resolve this problem if `conda update conda` fails?  Is there some clever work-around?

**Comment by rahulmanuwas** [comment]
Try to run `conda update conda` on command prompt (admin). It resolves the issue for me!

**Comment by kalefranz** [comment]
All of this should be fixed with conda 4.3.13.  If not, PLEASE let me know.  To get conda 4.3.13,

    conda update conda

If you installed conda/Anaconda as admin, then you'll need to also run `conda update conda` as admin.
